chord_metadata_service/chord/workflows/wdls/cbioportal_export.wdl

```diff
@@ -62,23 +57,17 @@ task katsu_dataset_export {
     # command block (tested with womtool-v78). Using triple angle braces made
     # interpolation more straightforward.
     command <<<
-        # Export results at export_path and returns http code 200 in case of success
-        RESPONSE=$(curl -X POST ~{true="" false="-k" validate_ssl} -s -w "%{http_code}" \
+        # Export results; Katsu returns a .tar.gz file
+        curl -X POST ~{true="" false="-k" validate_ssl} -s --fail-with-body \
             -H "Content-Type: application/json" \
             -H "Authorization: Bearer ~{token}" \
-            -d '{"format": "cbioportal", "object_type": "dataset", "object_id": "~{dataset_id}", "output_path": "~{run_dir}"}' \
-            "~{katsu_url}/private/export")
-
-        if [ "${RESPONSE}" != "204" ]
-        then
-            echo "Error: Metadata service replied with HTTP code ${RESPONSE}" 1>&2  # to stderr
-            exit 1
-        fi
-        echo ${RESPONSE}
```

Comment:
I think the workflow could fail silently now, no?

Reply:
I think it shouldn't because of the --fail-with-body, although it won't have the `Error: Metadata service replied with HTTP code` print now...